roles/bootstrap-os/tasks/bootstrap-debian.yml

```diff
@@ -20,9 +20,6 @@
   # This command should always run, even in check mode
   check_mode: false
   environment: {}
-  when:
-    - http_proxy is defined
-    - not skip_http_proxy_on_os_packages
```

Comment:
Instead of removing this block, can we avoid this issue by changing the order of `Add http_proxy to /etc/apt/apt.conf if http_proxy is defined` like
```
    - http_proxy is defined
    - not skip_http_proxy_on_os_packages
    - need_http_proxy.rc != 0
```
?
It is nice to avoid operating unnecessary tasks like the original one, and I'd like to find it if possible.

Reply:
The order cause not the error. Ansible-doc: When you have multiple conditions that all need to be true (that is, a logical and).  

The task  `- name: Check http::proxy in apt configuration files` will never be execute in this constellation (http_proxy set, skip_http_proxy_on_os_packages=true) and this caused the  error **need_http_proxy.rc != 0): 'dict object' has no attribute '**

Reply:
Thanks for your explanation.
I got it is necessary to remove the condition like this.

/lgtm